oak_restricted_kernel_sdk_proc_macro/src/lib.rs

```diff
@@ -62,11 +62,18 @@ fn process_entry_fn(entry_fn: ItemFn) -> TokenStream {
     let generated = quote! {
         #entry_fn
 
+        #[global_allocator]
```

Comment:
How obvious would the error be, if someone was already installing the allocator manually? Will they be able to figure it out?

Reply:
they would be. it's pretty legible.

```
error: cannot define multiple global allocators
  --> oak_functions_enclave_app/src/main.rs:33:1
   |
30 | / static ALLOCATOR: oak_restricted_kernel_sdk::utils::heap::LockedGrowableHeap =
31 | |     oak_restricted_kernel_sdk::utils::heap::LockedGrowableHeap::empty();
   | |________________________________________________________________________- previous global allocator defined here
32 |
33 |   #[entrypoint]
   |   ^^^^^^^^^^^^^ cannot define a new global allocator
   |
   = note: this error originates in the attribute macro `global_allocator` (in Nightly builds, run with -Z macro-backtrace for more info)
```